README.md

```diff
@@ -1,2 +1,74 @@
-# Capstone1
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
```

Comment:
- [ ] Kindly add the name of your project to your README to document your project professionally.